javascript/javascript1/week1/script3.js

```diff
@@ -0,0 +1,31 @@
+// for Peter's house
+let houseWidth = 8;
+let houseHeight = 10;
+let houseDepth = 10;
+let volumeInMeters = houseWidth * houseHeight * houseDepth;
+let gardenSizeInM2 = 100;
+let houseCalculatedPrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const PetersHouseActualPrice = 2500000;
+let priceDifference = PetersHouseActualPrice - houseCalculatedPrice;
+
+if(priceDifference > 0) {
+    console.log("Peter is paying " + Math.abs(priceDifference) + " too much for the house.");
+} else {
+    console.log("Peter is paying " + Math.abs(priceDifference) + " too little for the house.");
+}
```

Comment:
What is the case for if the price of the house is just right? What will be logged?

Reply:
Yes, I am adding one more condition for that. Didn't think about it :)